odl/tomo/geometry/parallel.py

```diff
@@ -57,8 +57,7 @@ def __init__(self, ndim, apart, detector, det_init_pos):
         detector : `Detector`
             The detector to use in this geometry
         det_init_pos : `array-like`
-            Initial position of the detector reference point. The zero
```

Comment:
It's not forbidden to put the detector into the origin. May not make too much sense physically but I see no reason to forbid it.